code/datums/elements/decals/blood.dm

```diff
@@ -13,6 +13,7 @@
 
 /datum/element/decal/blood/generate_appearance(_icon, _icon_state, _dir, _plane, _layer, _color, _alpha, _smoothing, source)
 	var/obj/item/I = source
+	ADD_KEEP_TOGETHER(I, "item_blood_overlay")
```

Comment:
This needs to `REMOVE_KEEP_TOGETHER` when the decal is detached (the blood is cleaned off)?

Reply:
@MrMelbert should be fine now, added to Detach()